app/helpers/votes_helper.rb

```diff
@@ -6,4 +6,12 @@ def voteable_has_user_vote?(voteable)
   def total_votes(voteable)
     voteable.votes.sum(:value)
   end
+
+  def user_vote_articles(arr)
```

Comment:
what is `arr` ?

Reply:
It's supposed to be an array of article IDs. I changed the name to be more readable.